src/component.ts

```diff
@@ -10,6 +10,15 @@ export class Component extends BaseSDK {
 		super({});
 		this._id = props.componentId;
 		this.type = "Component";
+		if (props?.manifestMethods?.length > 0) {
+			props.manifestMethods.forEach((method) => {
+				this[method.name] = (...args) =>
+					this._postMessageAsync(`COMPONENT_${method.name}`, {
+						id: this._id,
+						parameters: args
```

Comment:
why manifest methods alone having parameter key is there any specific reason ?

Reply:
Yes.. Since these are dynamic methods there can be n number of arguments for a function so we are spreading at function def and using them collectively while posting message